wrapt_test.go

```diff
@@ -0,0 +1,43 @@
+package wrapt_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/metrumresearchgroup/wrapt"
+)
+
+func Test_WrapT(tt *testing.T) {
+	t := wrapt.WrapT(tt)
+	t.RunFatal("positive assertion without failure", func(t *wrapt.T) {
+		t.A.True(true, "not true")
```

Comment:
Personally I think this would be worth calling out in the readme. The big benefit (to me) is the conceptual pairing of a test with its assertion and failure handler. 

So we can point out that `*T` is a struct that allows us to correlate an assertion -> Give some examples of assertions like the one here, and its handler

Reply:
massively updated readme.